app/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,158 @@
+    <string name="flip_camera">Odwróć aparat</string>
+    <string name="loading_camera">Wczytywanie aparatu</string>
+    <string name="flash_mode">Tryb lampy błyskowej</string>
+    <string name="capture_mode">Tryb przechwytywania</string>
+    <string name="start_value_timer">00:00</string>
+    <string name="zoom_in">Przybliż</string>
+    <string name="zoom_out">Oddal</string>
+    <string name="scan_result">Wynik Skanowania</string>
+    <string name="share_qr_text">Udostępnij Tekst QR</string>
+    <string name="copy_to_clipboard">Kopiuj do Schowka</string>
+    <string name="retake">Ponów</string>
+    <string name="play_preview">Odtwórz Podgląd</string>
+    <string name="toggle_button">Przycisk przełącznika</string>
+    <string name="aspect_ratio">Współczynnik Proporcji</string>
+    <string name="toggle_flash">Przełącz Lampę Błyskową</string>
+    <string name="grid_toggle">Przełącz Siatkę</string>
+    <string name="include_audio">Dołącz Dźwięk</string>
+    <string name="video_quality">Jakość Wideo</string>
+    <string name="optimize_for">Optymalizuj pod</string>
+    <string name="quality">Jakość</string>
+    <string name="latency">Opóźnienie</string>
+    <string name="preview">Podgląd</string>
+    <string name="file_info">Informacje o Pliku</string>
+    <string name="delete_file">Usuń plik</string>
+    <string name="edit">Edytuj</string>
+    <string name="self_illumination">Własne Oświetlenie</string>
+    <string name="focus_timeout">Limit Czasu Ostrości</string>
+    <string name="timer">Odliczanie</string>
+    <string name="cancel_timer">Anuluj Odliczanie</string>
+    <string name="camera_sounds">Dźwięki Aparatu</string>
+    <string name="video_capture_label">Nagrywaj Wideo</string>
+
+    <string name="camera">APARAT</string>
+    <string name="video">WIDEO</string>
+    <string name="portrait_mode">PORTRET</string>
+    <string name="hdr_mode">HDR</string>
+    <string name="night_mode">NOC</string>
+    <string name="face_retouch_mode">RETUSZ TWARZY</string>
+    <string name="auto_mode">AUTO</string>
+    <string name="qr_scan_mode">SKANER QR</string>
+    <string name="aspect_ratio_16_9">16:9</string>
+    <string name="aspect_ratio_4_3">4:3</string>
+    <string name="share_media">Udostępnij Media</string>
+    <string name="rec_without_audio">Rozpocznij nagrywanie bez dźwięku</string>
+    <string name="qr_scan_toggle">Przełącznik Skanera QR</string>
+    <string name="qr_scan">Kod QR</string>
+    <string name="data_matrix">Matryca Danych</string>
+    <string name="aztec_code">Aztec Code</string>
+    <string name="more_options">Więcej Opcji</string>
+    <string name="pdf_417">PDF 417</string>
+    <string name="more_settings">Więcej Ustawień</string>
+
+    <string name="general">Ogólne</string>
+
+    <string name="save_image_as_previewed">Zapisz obraz jako podgląd</string>
+    <string name="save_as_image_preview_desc">Zapisz obraz wyjściowy jako widoczny na podglądzie (poprawia inwersję)</string>
```

Comment:
*Zapisz obraz tak jak na podglądzie*

*tak, jak jest widoczny na podglądzie (poprawia odwrócenie)*

Kruci, nie umiem tego po polsku napisać tak aby wywnioskować że chodzi o przerzucanie selfie w pionie, ale to brzmi bardzo nieintuicyjnie

Reply:
Dzięki za wszystkie sugestie. Niestety, fotografem nie jestem - a nawet ze względu na brak wzroku nigdy nim nie będe, więc rzeczywiście, może kto inny powinien zająć się tłumaczeniem tej aplikacji? Póki co czekamy na powołanie translation teamu - <https://github.com/GrapheneOS/translation-team>